site/en/posts/iwd-2022-julia-enthoven.md

```diff
@@ -39,6 +39,10 @@ Eric, my co-founder, had left Google a few months before to start to work on his
 
 I tell that story for a few reasons. I think that a lot of people think that people who go into computer science have been coding since they were in middle school and that was definitely not the case for me. I took my first class the fall of my sophomore year. Even if you don’t have that much experience with it, you can grow your career as an engineer. Don't be afraid to start late.
 
+> I think that a lot of people think that people who go into computer science have been coding since they were in middle school and that was definitely not the case for me. I took my first class the fall of my sophomore year. Even if you don’t have that much experience with it, you can grow your career as an engineer. Don't be afraid to start late.
+
+Julia Enthoven, co-founder and CEO of Kapwing {.cite}
```

Comment:
How we do citations for comments has changed with our latest update. This should make them work as expected!

```suggestion
> I think that a lot of people think that people who go into computer science have been coding since they were in middle school and that was definitely not the case for me. I took my first class the fall of my sophomore year. Even if you don’t have that much experience with it, you can grow your career as an engineer. Don't be afraid to start late.
> {cite="Julia Enthoven, co-founder and CEO of Kapwing"}
```